packages/metadata-service/src/readers/reader.service.ts

```diff
@@ -22,11 +27,20 @@ export class ReaderService {
 
     const startedAt = new Date();
     let obj = undefined;
-    try {
-      const content = await this.ozbyReader.readData(isbn);
-      obj = this.ozbyReader.parseData(content);
-    } catch (e) {
-      console.error(`Identifier "${isbn}" not found!`);
+    let content = undefined;
+
+    const arrayOfServices = ['ozbyReader', 'chitaiGorodService'];
+
+    for (let i = 0; i <= arrayOfServices.length; i++) {
+      try {
+        content = await this[arrayOfServices[i]].readData(isbn);
```

Comment:
This uses sequential read instead of parallel - let's ignore it for today, but fix in the next PR.